### ISSUE TITLE
Tighten speaker bureau requirements

### ISSUE THREAD

**Issue by dankohn** [cncf/wordpress-site#94]
Please edit https://www.cncf.io/speakers/register/ to require that they include a bio and a picture. We can give existing folks ~3 months to comply.

**Comment by KimMcMahon** [comment]
@cjyabraham Keep other current required items. Add bio and a picture.

**Comment by cjyabraham** [comment]
@KimMcMahon there is no way to have them add a photo in that initial registration page.  When we discussed this previously we agreed on having a two step process: Have them register with just essential info, receive confirmation of acceptance, and then ask them to complete their profile.

**Comment by dankohn** [comment]
Can we leave them unpublished until they submit the rest of their information?

**Comment by KimMcMahon** [comment]
@cjyabraham - Love @dankohn suggestion. If so, it would be a 2 step process - register with minimal information, then to a screen to fill in the rest to "publish". LMK if this process will work and I'll update fields and language for both steps.

**Comment by dankohn** [comment]
The other key step would be for @KimMcMahon to manually approve them out of draft mode.

**Comment by cjyabraham** [comment]
I've added [bio](https://www.cncf.io/speakers/register/) and made it required.

**Comment by cjyabraham** [comment]
Ultimate Member [have replied](https://wordpress.org/support/topic/photo-upload-on-registration-form/#post-11936902) to say it's not possible to have the photo uploaded in the reg form.

I suggest not requiring the photo in order to be approved for the Speaker's Bureau but, instead, to have a strongly worded ask that people upload a profile photo in the "Approved" email they receive.  

As an additional option, we can choose to only list those who *have* uploaded a photo in the [Directory](https://www.cncf.io/speakers/), so we could say to people "You are approved but will not be listed in our Directory until you upload a photo."

Any thoughts?

**Comment by dankohn** [comment]
The latter. You won’t be in the directory until you upload a photo. Could we leave people in draft state until they’re manually approved?

**Comment by cjyabraham** [comment]
Yes.  They are technically in "Pending Review" state until they are manually approved.

**Comment by dankohn** [comment]
So, then it's as simple as not having CNCF staff approve submissions until they include an image.

**Comment by KimMcMahon** [comment]
Can we do a 2 step approval? The first step similar to what we have, then after I do the first approval, the email instructions say to fill out the rest (including picture) for approval and they is when they are on the website?

Or can we set up an “auto” approve with the first step. Takes them to the one to finish filling out their profile, then I review and I approve or deny?

**Comment by cjyabraham** [comment]
How it currently works is this:
1. User fills out the [registration form](https://www.cncf.io/speakers/register/)
2. User gets an email saying "Your account has to be manually reviewed."  They can not login or access their account at this time.
3. An admin goes in and approves the account.
4. The user gets an email "Your account is now approved."  They can now login and update their profile with an image etc.  Their profile is visible in [the directory](https://www.cncf.io/speakers/).  Any [gravatar](https://en.gravatar.com) associated with their email address is shown as their profile photo.

There is not much flexibility to modify this workflow.  

There is an option, however, to require a photo be uploaded before the profile is visible in the directory.  If we enable that option, however, we need to turn off the ability to use a gravatar for a user who has not uploaded a photo (it's a limitation of the plugin).  If we want to incentivize people to upload a photo, though, I suggest we activate that option and let users know in the email in step 4 that their profile won't show in the directory until they upload a photo.  

This will modify step 4 to be:

4. The user gets an email "Your account is now approved.  Please upload a photo to your profile in order to have it listed in the directory."  They can now login and update their profile with an image etc.  Their profile is visible in [the directory](https://www.cncf.io/speakers/) only after they upload a photo. 

Any thoughts?

**Comment by dankohn** [comment]
Could we instead automatically approve but not list their profile, and only
manually approve it after they upload their photo?
--
Dan Kohn <dan@linuxfoundation.org> +1-415-233-1000
Executive Director, Cloud Native Computing Foundation cncf.io
<https://www.cncf.io/>
dankohn.com <https://www.dankohn.com/> or book on my calendar: dankohn.com/c
<https://www.dankohn.com/c>

**Comment by cjyabraham** [comment]
We don't have that option within the constraints of the plugin.

**Comment by KimMcMahon** [comment]
@cjyabraham Please do the work for your option:
This will modify step 4 to be:

The user gets an email "Your account is now approved. Please upload a photo to your profile in order to have it listed in the directory." They can now login and update their profile with an image etc. Their profile is visible in the directory only after they upload a photo.

I'll work on the email they'll get.

**Comment by cjyabraham** [comment]
I've created #112 to get started on this.

**Comment by KimMcMahon** [comment]
Remaining for Kim: Emails each Speaker gets.

**Comment by cjyabraham** [comment]
@KimMcMahon is this issue still active?  If not, please close.

**Comment by KimMcMahon** [comment]
@cjyabraham This is still open. I need to update the emails.

**Comment by KimMcMahon** [comment]
Closing. New issue for the emails. I will manually check for compliance with our required items and deactivate profiles with communications to the speaker.
